RecoVertex/VertexPrimitives/interface/VertexState.h

```diff
@@ -11,42 +11,62 @@
  */
 
 class VertexState final : private  BasicVertexState::Proxy {
-
+  
   using Base =  BasicVertexState::Proxy;
   using BSVS =  BasicSingleVertexState;
-public:
-  VertexState(){}
+ public:
+  VertexState() :
+  Base ( new BSVS () ) {}
```

Comment:
why is this needed?
It looks like the initial design is the default constructed VertexState is invalid